cinemy/cinema/migrations/0013_create_periodic_tasks.py

```diff
@@ -0,0 +1,24 @@
+from django.db import migrations
+from django_q.tasks import schedule, Schedule
+
+
+def create_periodic_tasks(apps, schema_editor):
+    schedule(
+        func="cinema.tasks.tasks.mark_unconfirmed_reservations_as_expired",
```

Comment:
Where is this function? I can't seem to find this path in the repo.

Reply:
Ok, I found it in the other PR, but it's weird that it wasn't visible earlier. Why was this migration not included in the same PR? To me it makes more sense that way.

Reply:
This migration was created as part of an enhancement to the previous functionality in which the admin had to manually create the scheduled tasks.